engines/ospray/OSPRayModel.cpp

```diff
@@ -457,7 +456,7 @@ bool OSPRayModel::_commitTransferFunction()
 
 void OSPRayModel::_setBVHFlags()
 {
-    ospSet1i(_model, "dynamicMode", _bvhFlags.count(BVHFlag::dynamic));
+    ospSet1i(_model, "dynamicScene", _bvhFlags.count(BVHFlag::dynamic));
     ospSet1i(_model, "compactMode", _bvhFlags.count(BVHFlag::compact));
     ospSet1i(_model, "robustMode", _bvhFlags.count(BVHFlag::robust));
```

Comment:
Now one has the suffix prefix and the others are mode. I don't like that inconsistency.